serving_with_ipex_triton/README.md

```diff
@@ -0,0 +1,49 @@
+# Serving DenseNet models with IPEX and Triton Server
+
+## Description
+This sample provide code to integrate Intel® Extension for PyTorch (IPEX) with Triton Inference Server framework. This project provides custom python backend for IPEX and can be used as performance benchmark for DenseNet.
+
+## Preparation
+Make sure that Docker is installed on both host and client instance.
+Sample images from ImageNet dataset. 
+
+## Supported models
+Currently AI Inference samples support following Bert models finetuned on Squad dataset:
+- DenseNet121        - PyTorch+IPEX [DenseNet121](https://pytorch.org/hub/pytorch_vision_densenet/ "DenseNet121")
+
+## Possible run scenarios
+AI Inference samples allow user to run inference on localhost or on remote Triton Server Host. 
+By default config.properties is filled with localhost run option. 
+
+### Execution on localhost
+To build, start Docker containers, run tests, stop and do cleanup on localhost execute scripts in following order:
+
+`$ docker build -t tritonserver_custom -f Dockerfile.compose .`  - builds Docker image for Triton Server
+
+`$ bash start.sh`  - runs Docker containers for Triton Server Client and Host for model specified in config.properties
```

Comment:
it looks like only host server is being run in `start.sh`

Reply:
`config.properties` doesn't exist and not used

Reply:
Updated!